USAGE.md

```diff
@@ -85,17 +85,11 @@ nmcli c mod ${CONTAINER_RUNTIME_IF} ipv4.dns-search ${CONTAINER_DOMAIN}
 Add a static A record for the container hosting FreeIPA, Keycloak and Vault. Alternatively, adjust your `/etc/hosts` file accordingly:
```

Comment:
this line must be adjusted to fit the next one `sudo ./02-add-hosts.sh`. Something e.g. 
> To add a static A records to the `/etc/hosts` file for the containers hosting FreeIPA, Keycloak and Vault run

Reply:
Thanks for the correction.